pkg/microservice/aslan/core/environment/handler/service.go

```diff
@@ -65,10 +68,10 @@ func UpdateService(c *gin.Context) {
 	}
 
 	args := &service.SvcOptArgs{
-		EnvName:     c.Query("envName"),
-		ProductName: c.Param("productName"),
+		EnvName:     envName,
+		ProductName: projectName,
 		ServiceName: c.Param("serviceName"),
-		ServiceType: c.Param("serviceType"),
+		ServiceType: c.Query("serviceType"),
 		ServiceRev:  svcRev,
```

Comment:
c.query(name) write outside of the SvcOptArgs while c.query(serviceType) write outside